site/create-image.md

```diff
@@ -33,6 +34,9 @@ Usage: imagetool create [OPTIONS]
 | `--wdtArchive` | Path to the WDT archive file used by the WDT model.  |   |
 | `--wdtDomainHome` | Path to the `-domain_home` for WDT.  |   |
 | `--wdtDomainType` | WDT domain type. Supported values: `WLS`, `JRF`, `RestrictedJRF`  | `WLS`  |
+| `--wdtEncryptionKey` | Passphrase for WDT -use_encryption, that will be requested on STDIN. |   |
+| `--wdtEncryptionKeyEnv` | Passphrase for WDT -use_encryption, that is provided as an environment variable. |   |
+| `--wdtEncryptionKeyFile` | Passphrase for WDT -use_encryption, that is provided as a file. |   |
```

Comment:
-use_encryption, that -> -use_encryption that (remove the comma after -use_encryption in all three descriptions.)